content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst

```diff
@@ -246,19 +250,35 @@ manufacturing lead times and manufacturing security lead times.
 Manufacturing lead time
 -----------------------
 
-Configure the manufacturing lead time directly on the product form, by navigating to
-:menuselection:`Manufacturing app --> Products --> Products`, and select the desired product.
+Manufacturing lead times for products are configured from a product's Bill of Materials (BoM) form.
```

Comment:
will want to confirm with Sam, but "Bill of Materials form" may need some kind of additional formatting (GUI label or italics)? (not sure, tossing it out there)

Reply:
I dont think any formatting is necessary, as well as the capitalization isnt necessary either since BOM is an industry term and the form doesnt have a heading of "Bill of Materials"
```suggestion
Manufacturing lead times for products are configured from a product's bill of materials (BoM) form.
```